lib/github_api/params_hash.rb

```diff
@@ -9,7 +9,7 @@ class ParamsHash < DelegateClass(Hash)
     include Normalizer
     include MimeType
 
-    REQUEST_PARAMS = [:accept, :media, :data, :raw, :content_type, :headers]
+    REQUEST_PARAMS = [:accept, :media, :data, :raw, :content_type, :headers, :options]
```

Comment:
Sorry I have probably mislead you with my suggestion as I haven't worked on implementing apis for a while now. However, I have found [reviews api](https://github.com/piotrmurach/github/blob/master/lib/github_api/client/pull_requests/reviews.rb) as an example with preview accept header.